theseus/labs/lie_functional/se3.py

```diff
@@ -118,16 +120,16 @@ def randn(
         device=device,
         requires_grad=requires_grad,
     )
-    return torch.cat((rotation, translation), dim=2)
+    ret = torch.cat((rotation, translation), dim=2)
+    ret.requires_grad_(requires_grad)
```

Comment:
You could similarly remove the two `requires_grad=requires_grad,` lines for `randn`.

Reply:
Ah sorry, I thought I did. Will fix.

Reply:
Fixed.